in-toto-spec.md

```diff
@@ -1685,11 +1686,11 @@ link metadata:
        "src/foo.c": { "sha256": "2a0ffef5e9709e6164c629e8b31bae0d..."},
        "vcs.log": { "sha256": "e64589ab156f325a4ab2bc5d532737d5a7..."}
     },
-    "return-value": "0",
+    "return-value": 0,
```

Comment:
This seems like a remainder from when `return-value` was a 1st level field of `signed`. I suggest to remove this.

Reply:
ah good catch! I didn't even realize that there is a return-value outside of the actual byproducts object.